pkg/spec/namespaces.go

```diff
@@ -216,6 +216,9 @@ func (c *CgroupConfig) ToCreateOptions(runtime *libpod.Runtime) ([]libpod.CtrCre
 	if c.Cgroups == "disabled" {
```

Comment:
IMO we should disable this and remove WithNoCgroups, and move everything to use WithCgroupsMode

Reply:
We'll need to retain NoCgroups in the database and keep supporting it there for legacy reasons, but we should move all new containers over to use the new version

Reply:
If we are depracatng it, then remove it from the docs and hide the option.  No reason for people to continue to use it.

Reply:
It should continue working - we're just changing the way we store it in the database

Reply:
But I thought we were discouraging an option?